spec/twilio/twilio-reply-spec.js

```diff
@@ -28,7 +28,7 @@ describe('Twilio Reply', () => {
 
   it('sends text messages as a string', done => {
     https.request.pipe(callOptions => {
-      expect(qs.parse(callOptions.body)).toEqual({
+      expect(JSON.parse(JSON.stringify(qs.parse(callOptions.body)))).toEqual({
```

Comment:
@stojanovic @gojko this is because of https://nodejs.org/api/querystring.html#querystring_querystring_parse_str_sep_eq_options
>Note: The object returned by the querystring.parse() method **does not** prototypically extend from the JavaScript Object